src/ontology/Modules/occo_terms.owl

```diff
@@ -464,6 +513,249 @@
     
 
 
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000118 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000118">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/IAO_0000310"/>
+        <obo:IAO_0000115>A document issued by a third party bearing relevant authority, that is designed to be about an agent&apos;s competence, qualifications, or suitability with respect to specific skills, abilities, knowledge, or character. </obo:IAO_0000115>
+        <obo:IAO_0000117>Bill Duncan</obo:IAO_0000117>
+        <obo:IAO_0000117>Bill Hogan</obo:IAO_0000117>
+        <obo:IAO_0000117>Finn Wilson</obo:IAO_0000117>
+        <obo:IAO_0000117>Giacomo De Colle</obo:IAO_0000117>
+        <obo:IAO_0000117>Jie Zheng</obo:IAO_0000117>
+        <obo:IAO_0000117>John Beverley</obo:IAO_0000117>
+        <obo:IAO_0000117>Matthew Diller</obo:IAO_0000117>
+        <obo:IAO_0000117>Oliver He</obo:IAO_0000117>
+        <obo:IAO_0000117>Robin McGill</obo:IAO_0000117>
+        <obo:IAO_0000117>Sam Smith</obo:IAO_0000117>
+        <obo:IAO_0000119>https://storage.googleapis.com/wzukusers/user-12947767/documents/2c18c4f456ce4f6eab176c0983f75791/FOIS_2023_paper_920.pdf</obo:IAO_0000119>
+        <obo:IAO_0000119>https://www.degruyter.com/document/doi/10.1515/jso-2019-0034/html</obo:IAO_0000119>
+        <obo:IAO_0000233>https://github.com/Occupation-Ontology/OccO/issues/31</obo:IAO_0000233>
+        <rdfs:label>credential</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000119 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000119">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OCCO_00000118"/>
+        <obo:IAO_0000115>A crediential that is associated with an occupation.</obo:IAO_0000115>
```

Comment:
Definition reads "A credential that is designed to be about an organism's competence or qualifications with respect to an occupation"